include/types.h

```diff
@@ -143,14 +143,18 @@ struct KirbyState {
     u32 unk8C;
     u32 powerup; //0x90
     u32 currentInhale; //0x94
-    u32 unk98;
-    u32 unk9C;
-    u32 unkA0;
+    u32 firstInhale; //0x98
```

Comment:
Thanks for the PR! I am noticing some references to `gKirbyState.unkXX` fields that havent been renamed to the new ones. You can change those using `./rename_sym.sh gKirbyState.unkXX gKirbyState.newField` if you have the repo cloned

Reply:
Done. Still getting used to this (and git as well)